susemanager-utils/susemanager-sls/salt/switch_to_bundle/mgr_switch_to_venv_minion.sls

```diff
@@ -0,0 +1,129 @@
+{%- if grains.saltversion == '3000' and not grains.get('is_bundle') -%}
```

Comment:
We would need to exclude SSH minions here, as there might be Salt 3000 SSH minions (py27-compat-salt) still in 4.2, as it does not yet use Salt Bundle to drive Salt SSH actions.

Reply:
Yeah, it should, as `py27-compat-salt` is actually Salt 3000.3.

But, in order to exclude SSH minions from this, I think we should use:

```jinja
{%- if salt['pillar.get']('contact_method') not in ['ssh-push', 'ssh-push-tunnel'] %}
...
{% endif %}
```

As we generally do for excluding SSH minions in other SLS files, like: https://github.com/uyuni-project/uyuni/blob/master/susemanager-utils/susemanager-sls/salt/services/salt-minion.sls#L10